token-lending/program/src/processor.rs

```diff
@@ -419,6 +419,26 @@ fn _refresh_reserve<'a>(
     Ok(())
 }
 
+/// Lite version of refresh_reserve that should be used when the oracle price doesn't need to be updated
+/// BE CAREFUL WHEN USING THIS
+fn _refresh_reserve_lite<'a>(
```

Comment:
This is an internal function, which is good. If it were publicly callable could result in some weird stuff (e.g. attacker might be able to use a really stale price to liquidate someone since this marks the reserve as up to date).

Reply:
yeah 100%

Reply:
By internal I mean it's not invokable since there's no `process_refresh_reserve_lite` matcher in `process_instruction`.